src/librustc_passes/check_attr.rs

```diff
@@ -21,26 +22,6 @@ use rustc_session::parse::feature_err;
 use rustc_span::symbol::sym;
 use rustc_span::Span;
 
-fn target_from_impl_item<'tcx>(tcx: TyCtxt<'tcx>, impl_item: &hir::ImplItem<'_>) -> Target {
```

Comment:
This is in the same crate, so you could just make it public instead of moving it.

Reply:
I thought it was better to be able to consistently use `Target::from_[type]`